sdcm/cluster.py

```diff
@@ -2606,6 +2606,7 @@ def run_nodetool(self, sub_cmd, args="", options="", timeout=None,
             try:
                 # HACK: temporary hack to use libssh2 only for nodetool command
                 remote_class = RemoteCmdRunnerBase.remoter_classes.get('libssh2')
+                self.ssh_login_info["hostname"] = self.external_address
```

Comment:
Wdyt about changing it in parameters for `remote_class` to avoid such side-effect in this function?

Smth like following:

```python
remoter = remote_class(**self.ssh_login_info | {"hostname": self.external_address})
```

Reply:
fixed